notebooks/langchain-pipeline-rosa.ipynb

```diff
@@ -0,0 +1,894 @@
+{
```

Comment:
<p>I feel we can add some conclusion for this nb as well? Wdyt? Otherwise /lgtm </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>+1 </p><p>If we can summarize the cool experiments with small and large chunks, that'd be good to see.</p>

Reply:
Thanks for the feedback. I expanded some comments to hopefully cover what you expected, and added a conclusion to both notebooks.

PTAL :)